lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -323,16 +323,20 @@ def test_access_control(self):
 
 
 @ddt.ddt
-@httpretty.activate
 class GetThreadListTest(CommentsServiceMockMixin, ModuleStoreTestCase):
     """Test for get_thread_list"""
     def setUp(self):
         super(GetThreadListTest, self).setUp()
+        httpretty.enable()
```

Comment:
call `httpretty.reset()` too, to avoid unexpected side effects from previous tests.